SampleApp/src/PortAudioMicrophoneWrapper.cpp

```diff
@@ -79,6 +111,17 @@ bool PortAudioMicrophoneWrapper::initialize() {
         ACSDK_CRITICAL(LX("Failed to create stream writer"));
         return false;
     }
+#ifdef XMOS_AVS_TESTS
+    if (PortAudioMicrophoneWrapper::m_isFileStream) {
+        m_fileStream = new std::ifstream{"/tmp/in.raw", std::ios::binary};
```

Comment:
I know it's only a temp file and it's only for testing, but is it worth making this file name a variable so that in the unforeseeable event that you ever need to stream from somewhere else, you can..? Like, have it default to /tmp/in.raw but overwritable with an environment variable? Just a thought...

Reply:
It is a good idea, but I am not sure it is very practical. The tests are normally run from 
https://github.com/xmos/avs_lab_tests/blob/master/get_amazon_responses.py
and this script assumes the audio file is located there and it takes care of pre-processing the audio file among other things. I don't think it will be easy to run the AVS tests without that script.
What type of use cases do you think this feature can be used?